fnproject/fn
# remote runner support for trace exporters (#1538)

Handling docker metrics in a different issue

adding span attrib support for remote runner metrics (docker wait time, etc. in runner_client)
adding ochttp filter to group invoke calls in server.go (removes traces from /, /version paths)
changing up the grpc callfinished for display on jaeger (adding image and docker wait fields)
Adding span.setstatus to some spans

- Link to issue this resolves
https://github.com/fnproject/fn/issues/1538

- What I did
- How I did it
Added image name and docker wait duration fields to callfinished grpc response sent to runner_client, span annotations, attributes for visibility/debugging

- How to verify it
Set a jaeger/zipkin url, invoke a function on a remote runner and check attributes in new span receive_from_runner
![image](https://user-images.githubusercontent.com/25177182/62658689-37edce80-b91e-11e9-84ba-e2397d960aff.png)

- One line description for the changelog
remote runner support for jaeger (#1538)
